README.md

```diff
@@ -61,41 +60,41 @@ optional arguments:
                         Event sampling frequency of perf-record, in Hz.
   --perf-time PERF_TIME
                         Time period of perf recording, in seconds.
-  -l, --log             Enable to include log files in output, PD/TiDB/TiKV
-                        logs are included by default.
-  --syslog              Enable to include system log in output, will be
-                        ignored if -l/--log is not set. This may significantly
-                        increase output size.
-  --config-file         Enable to include various config files in output,
+  -l, --log             Collect log files in output. PD/TiDB/TiKV logs are
+                        included by default.
+  --syslog              Collect the system log in output. This may
+                        significantly increase output size. If `-l/--log` is
+                        not set, the system log will be ignored.
+  --config-file         Collect various configuration files in output,
                         disabled by default.
-  --pd-host PD_HOST     The host of PD server. Default to localhost.
-  --pd-port PD_PORT     The port of PD API service, default to 2379.
+  --pd-host PD_HOST     The host of the PD server. `localhost` by default.
+  --pd-port PD_PORT     The port of PD API service, `2379` by default.
 
 Note that some arguments would decrease system performance.
 ```
 
-If calling `insight.py` with no argument provided, following information are collected and saved to `data` directory of current working directory:
+If `insight.py` is called with no argument provided, the following information is collected and saved to `data` directory in the current working directory:
 
- - `collector`, providing system information, program versions, NTP status, partition statics and process statics of TiDB modules (if present), placed under `collector` sub-directory.
- - PD API output of `config`, `diagnose`, `health`, `hotspot`, `labels`, `members`, `operators`, `regions`, `regionstats`, `schedulers` and `status`, placed under `pdctl` sub-directory in JSON format.
+ - `collector`, which provids the system information, program versions, NTP status, partition statistics and process statistics of TiDB modules (if present), placed in the `collector` sub-directory.
```

Comment:
`collector` which **provides** the system information, program versions, NTP status, partition statistics and process statistics of TiDB modules (if present), placed in the `collector` sub-directory.